modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItNoThreadsLeftTest.java

```diff
@@ -123,7 +123,6 @@ private static Table createTable(Ignite node, String tableName) {
 
     /**
      * Get a set threads.
-     * TODO: IGNITE-15161. Filter will be removed after the stopping for all components is implemented.
```

Comment:
but we still have the filter

Reply:
Yes, we have. I think that initial comment is not quite correct. For instance, we cannot guarantee that threads from the common pool will be shut down.